setup-python-runner/action.yaml

```diff
@@ -0,0 +1,66 @@
+name: Setup Python runner
+description: Set up a GitHub runner for Starcraft Python workflows
+
+inputs:
+  lxd-channel:
+    type: string
+    default: 5.21/candidate
+
+runs:
+  using: composite
+  steps:
+    - name: Start apt update
+      if: runner.os == 'Linux'
+      id: apt
+      shell: bash
+      run: |
+        nohup sudo apt-get update > /dev/null &
+        echo "apt-pid=$!" >> "${GITHUB_OUTPUT}"
+    - name: Setup LXD
+      if: ${{ runner.os == 'Linux' }}
+      uses: canonical/setup-lxd@v0.1.2
+      with:
+        channel: ${{ inputs.lxd-channel }}
+    - name: Fetch source
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+    # On mac, set up homebrew caching since we use homebrew to install dependencies.
+    - name: Set up Homebrew
+      if: runner.os == 'macOS'
+      id: set-up-homebrew
+      uses: Homebrew/actions/setup-homebrew@master
+    - name: Cache Homebrew Bundler RubyGems
+      if: runner.os == 'macOS'
+      id: cache
+      uses: actions/cache@v3
+      with:
+        path: ${{ steps.set-up-homebrew.outputs.gems-path }}
+        key: ${{ runner.os }}-rubygems-${{ steps.set-up-homebrew.outputs.gems-hash }}
+        restore-keys: ${{ runner.os }}-rubygems-
+    - name: Install Homebrew Bundler RubyGems
+      if: steps.cache.outputs.cache-hit != 'true' && runner.os == 'macOS'
+      shell: bash
+      run: brew install-bundler-gems
+    - name: Hash runner info
+      id: runner-info
+      shell: bash
+      run: |
+        echo "cache-hash=$(echo "${{ toJSON(matrix.platform) }}" | sha1sum | cut -f1 -d' ')" >> "${GITHUB_OUTPUT}"
+    - name: Set up uv with caching
+      id: setup-uv
```

Comment:
Previously the `python-version` key was used for this `setup-uv` action (see https://github.com/canonical/starflow/pull/58/files#diff-5405a6a14181df246119f41da914c6b91501ad8fa55720ad5e0dbdcea40de723L159). It was lost in this change. 

What do you think of adding it back as an input for the `setup-python-runner` action?